services/core-web/src/components/noticeOfWork/applications/referals/NOWApplicationReviews.js

```diff
@@ -0,0 +1,317 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { Button, Row, Col, notification } from "antd";
+import { openModal, closeModal } from "@/actions/modalActions";
+import {
+  getNowDocumentDownloadToken,
+  getDocumentDownloadToken,
+} from "@/utils/actionlessNetworkCalls";
+import { modalConfig } from "@/components/modalContent/config";
+import CustomPropTypes from "@/customPropTypes";
+import * as Permission from "@/constants/permissions";
+import AuthorizationWrapper from "@/components/common/wrappers/AuthorizationWrapper";
+import AddButton from "@/components/common/AddButton";
+import {
+  createNoticeOfWorkApplicationReview,
+  fetchNoticeOfWorkApplicationReviews,
+  deleteNoticeOfWorkApplicationReview,
+  updateNoticeOfWorkApplicationReview,
+  deleteNoticeOfWorkApplicationReviewDocument,
+  setNoticeOfWorkApplicationDocumentDownloadState,
+} from "@/actionCreators/noticeOfWorkActionCreator";
+import { fetchNoticeOfWorkApplicationReviewTypes } from "@/actionCreators/staticContentActionCreator";
+import { getNoticeOfWorkReviews } from "@/selectors/noticeOfWorkSelectors";
+import { getDropdownNoticeOfWorkApplicationReviewTypeOptions } from "@/selectors/staticContentSelectors";
+import NOWApplicationReviewsTable from "@/components/noticeOfWork/applications/referals/NOWApplicationReviewsTable";
+
+/**
+ * @constant ReviewNOWApplication renders edit/view for the NoW Application review step
+ */
+
+const propTypes = {
+  // eslint-disable-next-line
+  noticeOfWorkGuid: PropTypes.string.isRequired,
+  mineGuid: PropTypes.string.isRequired,
+  // eslint-disable-next-line
+  noticeOfWorkReviews: PropTypes.arrayOf(CustomPropTypes.NOWApplicationReview).isRequired,
+  coreDocuments: PropTypes.arrayOf(PropTypes.objectOf(PropTypes.any)).isRequired,
+  submissionDocuments: PropTypes.arrayOf(PropTypes.objectOf(PropTypes.any)).isRequired,
+  noticeOfWorkReviewTypes: CustomPropTypes.options.isRequired,
+
+  openModal: PropTypes.func.isRequired,
+  closeModal: PropTypes.func.isRequired,
+  createNoticeOfWorkApplicationReview: PropTypes.func.isRequired,
+  fetchNoticeOfWorkApplicationReviews: PropTypes.func.isRequired,
+  fetchNoticeOfWorkApplicationReviewTypes: PropTypes.func.isRequired,
+  updateNoticeOfWorkApplicationReview: PropTypes.func.isRequired,
+  deleteNoticeOfWorkApplicationReview: PropTypes.func.isRequired,
+  deleteNoticeOfWorkApplicationReviewDocument: PropTypes.func.isRequired,
+  setNoticeOfWorkApplicationDocumentDownloadState: PropTypes.func.isRequired,
+};
+
+const defaultProps = {};
+
+export class NOWApplicationReviews extends Component {
+  state = { cancelDownload: false };
+  componentDidMount() {
+    this.props.fetchNoticeOfWorkApplicationReviews(this.props.noticeOfWorkGuid);
+    this.props.fetchNoticeOfWorkApplicationReviewTypes();
```

Comment:
This should be getting fetched in Ryans new static-content update. should be able to remove from here.

Reply:
removed.